src/collection-view.js

```diff
@@ -712,15 +714,15 @@ const CollectionView = Backbone.View.extend({
 
   // Destroy the child views that this collection view is holding on to, if any
   _destroyChildren(options) {
-    const childViews = this.children.map(_.identity);
+    const childModels = this.children.map((view) => {
+      return view.model;
+    });
 
-    if (childViews.length) {
+    if (childModels.length) {
```

Comment:
this might as well be
```js
_destroyChildren(options) {
  if (this.children.length) {
    this.triggerMethod('before:destroy:children', this);
    const childModels = this.children.map('model');
    this._removeChildModels(childModels, options);
    this.triggerMethod('destroy:children', this);
  }
}
```